docs/connector-development/config-based/low-code-cdk-overview.md

```diff
@@ -1,11 +1,29 @@
 # Low-code connector development
 
-Airbyte’s low-code framework enables you to build source connectors for REST APIs by modifying boilerplate YAML files.
+Airbyte’s low-code framework enables you to build source connectors for REST APIs via a connector builder UI or via modifying boilerplate YAML files via terminal or text editor.
```

Comment:
```suggestion
Airbyte’s low-code framework enables you to build source connectors for REST APIs via a [connector builder UI](connector-builder-ui.md) or by modifying boilerplate YAML files via terminal or text editor.
```
Link to the connector builder UI page, and remove one of the "via"s since there were three :p

Reply:
Or is there a reason you didn't want to link to that page here, and only link to it in the Connector Builder UI section down below?

Reply:
no particular reason, doesn't hurt to add more links to it